module/applications/dice/roll-configuration-dialog.mjs

```diff
@@ -325,6 +314,21 @@ export default class RollConfigurationDialog extends Dialog5e {
 
     this.options.buildConfig?.(this.config, config, formData, index);
 
+    /**
+     * A hook event that fires when a roll config is built using the roll prompt. Multiple hooks may be called depending
+     * on the rolling method (e.g. `dnd5e.buildSkillRollConfig`, `dnd5e.buildAbilityCheckRollConfig`,
+     * `dnd5e.buildRollConfig`).
+     * @function dnd5e.buildRollConfig
+     * @memberof hookEvents
+     * @param {RollConfigurationDialog|object} app  Roll configuration dialog or object that simulates it.
+     * @param {BasicRollConfiguration} config       Roll configuration data.
+     * @param {FormDataExtended} [formData]         Any data entered into the rolling prompt.
+     * @param {number} index                        Index of the roll within all rolls being prepared.
+     */
+    for ( const hookName of this.#config.hookNames ?? [""] ) {
+      Hooks.callAll(`dnd5e.build${hookName.capitalize()}RollConfig`, this, config, formData, index);
```

Comment:
Let's add a new hook here instead called something like `postRollConfig`, with a signature of `config, index, options`, where `options` has `app` and `formData`.